# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1436492649 14400
#      Thu Jul 09 21:44:09 2015 -0400
# Branch stable
# Node ID 2ec049e50ed8541cd37895b0554a98540993eb6a
# Parent  3300cb76cccdeaf47b9d8d589bd248e6c495778f
doc: Fix backslash characters in docstrings

* utils.cc (Fisindex), beep.m: Use @xbackslashchar to produce backslashes in
docstrings consistently and be compatible with Texinfo 6.

diff --git a/libinterp/corefcn/utils.cc b/libinterp/corefcn/utils.cc
--- a/libinterp/corefcn/utils.cc
+++ b/libinterp/corefcn/utils.cc
@@ -1374,17 +1374,17 @@ Valid indices are either positive intege
 type), or logical arrays.\n\
 \n\
 If present, @var{n} specifies the maximum extent of the dimension to be\n\
 indexed.  When possible the internal result is cached so that subsequent\n\
 indexing using @var{ind} will not perform the check again.\n\
 \n\
 Implementation Note: Strings are first converted to double values before the\n\
 checks for valid indices are made.  Unless a string contains the NULL\n\
-character @nospell{\"\\0\"}, it will always be a valid index.\n\
+character @nospell{\"@xbackslashchar{}0\"}, it will always be a valid index.\n\
 @end deftypefn")
 {
   octave_value retval;
   int nargin = args.length ();
   octave_idx_type n = 0;
 
   if (nargin == 2)
     n = args(1).idx_type_value ();
diff --git a/scripts/io/beep.m b/scripts/io/beep.m
--- a/scripts/io/beep.m
+++ b/scripts/io/beep.m
@@ -15,19 +15,19 @@
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} beep ()
 ## Produce a beep from the speaker (or visual bell).
 ##
-## This function sends the alarm character @qcode{"\a"} to the terminal.
-## Depending on the user's configuration this may produce an audible beep,
-## a visual bell, or nothing at all.
+## This function sends the alarm character @qcode{"@xbackslashchar{}a"} to
+## the terminal. Depending on the user's configuration this may produce an
+## audible beep, a visual bell, or nothing at all.
 ## @seealso{puts, fputs, printf, fprintf}
 ## @end deftypefn
 
 ## Author: jwe
 
 function beep ()
 
   if (nargin != 0)
